gruntfile.js

```diff
@@ -34,12 +34,12 @@ module.exports = function (grunt) {
                 tsconfig: './shared/AppInsightsCore/tsconfig.json'
             },
             coretest: {
-                tsconfig: './shared/AppInsightsCore/src/JavaScriptSDK.Tests/tsconfig.json',
```

Comment:
I moved the core tests out from the src folder, so they are now in the same place as all of the other components